### ISSUE TITLE
Executable including transformers module crashes after few seconds

### ISSUE THREAD

**Issue by Gition** [Nuitka/Nuitka#2774]
Hi,

I attempted to convert a script  into an executable in order to decrease execution time.
Throughout the compilation process, everything seems fine except for a few warnings. However, the executables crash after a few seconds. Based on my assessment and my testes, I believe that the transformers module is the source of this issue.

Can you help me to resolve this issue, please ?
Regards.
Ion

**This is my environment:**
Nuitka version: 2.1.2, non comercial
Python: 3.11.4 (tags/v3.11.4:d2340ef, Jun  7 2023, 05:45:37) [MSC v.1934 64 bit (AMD64)]
Flavor: CPython Official
Executable: C:\prgFiles\Python\Python311\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: ~\AppData\Local\Nuitka\Nuitka\Cache\downloads\gcc\x86_64\13.2.0-16.0.6-11.0.1-msvcrt-r1\mingw64\bin\gcc.exe (gcc 13.2.0).
Tranformers module: 4.30.2
Numpy module: 1.23.5
Pandas module: 2.0.2
Scipy module:1.10.1
Torch module: 2.0.1

**This is my command line:**
py -m nuitka --follow-imports  --noinclude-default-mode=warning  --noinclude-numba-mode=nofollow --module-parameter=numba-disable-jit=yes  --include-module=transformers --include-module=tensorflow --include-package=numpy --include-package=pandas --assume-yes-for-downloads --standalone --module-parameter=torch-disable-jit=yes --include-data-dir=../MODAI_LACHINE_EST/pretrained_tokenizer=pretrained_tokenizer --include-data-dir=../MODAI_LACHINE_EST/source_tokenizer=source_tokenizer  MyScript.py

**Messages :**
Nuitka-Options: Used command line options: --follow-imports --noinclude-default-mode=warning --noinclude-numba-mode=nofollow
Nuitka-Options: --module-parameter=numba-disable-jit=yes --include-module=transformers --include-module=tensorflow
Nuitka-Options: --include-package=numpy --include-package=pandas --assume-yes-for-downloads --standalone
Nuitka-Options: --module-parameter=torch-disable-jit=yes
Nuitka-Options: --include-data-dir=../MODAI_LACHINE_EST/pretrained_tokenizer=pretrained_tokenizer
Nuitka-Options: --include-data-dir=../MODAI_LACHINE_EST/source_tokenizer=source_tokenizer MyScript.py
Nuitka-Options: Following all imports is the default for standalone mode and need not be specified.
Nuitka: Starting Python compilation with Nuitka '2.1.2' on Python '3.11' commercial grade 'not installed'.
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'doctest' (intending to avoid 'unittest') in
Nuitka-Plugins:WARNING: 'numpy.testing._private.nosetester' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\testing\_private\nosetester.py:413') encountered. It may
Nuitka-Plugins:WARNING: slow down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'cython' (intending to avoid 'setuptools') in
Nuitka-Plugins:WARNING: 'numpy.random.tests.test_extending' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\random\tests\test_extending.py:29') encountered. It may
Nuitka-Plugins:WARNING: slow down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'Cython' (intending to avoid 'setuptools') in
Nuitka-Plugins:WARNING: 'numpy.random.tests.test_extending' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\random\tests\test_extending.py:30') encountered. It may
Nuitka-Plugins:WARNING: slow down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'setuptools' in 'numpy.distutils.core' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\distutils\core.py:8') encountered. It may slow down
Nuitka-Plugins:WARNING: compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'setuptools' in 'numpy.distutils.command.install' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\distutils\command\install.py:3') encountered. It may slow
Nuitka-Plugins:WARNING: down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'setuptools' in 'numpy.distutils.command.develop' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\distutils\command\develop.py:6') encountered. It may slow
Nuitka-Plugins:WARNING: down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka-Plugins:WARNING: anti-bloat: Undesirable import of 'setuptools' in 'numpy.distutils.command.bdist_rpm' (at
Nuitka-Plugins:WARNING: 'C:\prgFiles\Python\Python311\Lib\site-packages\numpy\distutils\command\bdist_rpm.py:4') encountered. It may
Nuitka-Plugins:WARNING: slow down compilation.
Nuitka-Plugins:WARNING:     Complex topic! More information can be found at https://nuitka.net/info/unwanted-module.html
Nuitka: Completed Python level compilation and optimization.
Nuitka: Generating source code for C backend compiler.
Nuitka: Running data composer tool for optimal constant value handling.
Nuitka: Running C compilation via Scons.
Nuitka-Scons: Backend C compiler: gcc (gcc 13.2.0).
Nuitka-Scons: Backend linking program with 8230 files (no progress information available for this stage).
Nuitka-Scons: Compiled 8190 C files using ccache.
Nuitka-Scons: Cached C files (using ccache) with result 'cache hit': 4595
Nuitka-Scons: Cached C files (using ccache) with result 'cache miss': 3595
Nuitka-Options: Included data file 'pretrained_tokenizer\config.json' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included data file 'pretrained_tokenizer\merges.txt' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included data file 'pretrained_tokenizer\vocab.json' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included data file 'pretrained_tokenizer\pretrained_tensors\attention_mask.pt' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included data file 'pretrained_tokenizer\pretrained_tensors\input_ids.pt' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included data file 'pretrained_tokenizer\pretrained_tensors\labels.pt' due to specified data dir
Nuitka-Options: '../MODAI_LACHINE_EST/pretrained_tokenizer' on command line.
Nuitka-Options: Included 82 data files due to specified data dir '../MODAI_LACHINE_EST/source_tokenizer' on command line.
Nuitka-Plugins:data-files: Included data file 'certifi\cacert.pem' due to package data for 'certifi'.
Nuitka-Plugins:data-files: Included data file 'grpc\_cython\_credentials\roots.pem' due to package data for 'grpc'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\html.tpl' due to package data directory 'templates' for
Nuitka-Plugins:data-files: 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\html_style.tpl' due to package data directory 'templates'
Nuitka-Plugins:data-files: for 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\html_table.tpl' due to package data directory 'templates'
Nuitka-Plugins:data-files: for 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\latex.tpl' due to package data directory 'templates' for
Nuitka-Plugins:data-files: 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\latex_longtable.tpl' due to package data directory
Nuitka-Plugins:data-files: 'templates' for 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\latex_table.tpl' due to package data directory 'templates'
Nuitka-Plugins:data-files: for 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included data file 'pandas\io\formats\templates\string.tpl' due to package data directory 'templates' for
Nuitka-Plugins:data-files: 'pandas.io.formats'.
Nuitka-Plugins:data-files: Included 602 data files due to package data directory 'zoneinfo' for 'pytz'.
Nuitka-Plugins:data-files: Included data file 'scipy\stats\_sobol_direction_numbers.npz' due to package data for 'scipy'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\boston_house_prices.csv' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\breast_cancer.csv' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\diabetes_data_raw.csv.gz' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\diabetes_target.csv.gz' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\digits.csv.gz' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\iris.csv' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\linnerud_exercise.csv' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\linnerud_physiological.csv' due to package data directory 'data'
Nuitka-Plugins:data-files: for 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included data file 'sklearn\datasets\data\wine_data.csv' due to package data directory 'data' for
Nuitka-Plugins:data-files: 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included 13 data files due to package data directory 'descr' for 'sklearn.datasets'.
Nuitka-Plugins:data-files: Included 8101 data files due to package data directory 'include' for 'torch'.
Nuitka-Plugins:data-files: Included 602 data files due to package data directory '.' for 'tzdata.zoneinfo'.
Nuitka-Plugins:dll-files: Found 1 file DLLs from numpy installation.
Nuitka-Plugins:dll-files: Found 2 files DLLs from sklearn._distributor_init installation.
Nuitka-Plugins:dll-files: Found 1 file DLLs from tensorflow.lite.experimental.microfrontend.python.ops.audio_microfrontend_op
Nuitka-Plugins:dll-files: installation.
Nuitka-Plugins:dll-files: Found 11 files DLLs from torch installation.
Nuitka: Keeping build directory 'MyScript.build'.
Nuitka: Successfully created 'MyScript.dist\MyScript.exe'.

**Error message:**
![Message](https://github.com/Nuitka/Nuitka/assets/13418475/f728661e-aae9-478f-a2f2-f9797ba3658a)

**Comment by kayhayen** [comment]
Yes, looking at screenshots is how we solve that. Without a minimal reproducer in the form of a Python script, there is nothing to do.

**Comment by Gition** [comment]
I checked in MyScript.exe.18808.dmp file and I found this:
Exception Code: 0xc0000005
Exception information: The thread tried to read or write to a virtual adress for wich it odes not have the appropriate access.
Heap Information: Mot present

**Comment by kayhayen** [comment]
Your use of `--include-module` is also relatively dumb, compile all test code of all these modules into the result, and then all test frameworks on the planet, not a good job.

**Comment by Gition** [comment]
I placed a message found in MyScript.exe.18808.dmp (C:\Users\user_name\AppData\Local\CrashDumps) file.
It seems that the error is caused by writing or reading in a virtual address without the appropriate access.

**Comment by kayhayen** [comment]
In the template, we have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that we do not want to be left guessing. For you it's easy to erase the issue template and expect us to ask you questions only as necessary, but we cannot do that. We would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. Therefore we are  likely just deleting the issue, as it serves as a bad example and detraction for searches.  And even worse, it might keep good reporters from reporting as is this "already known."
